src/main/java/com/Saurabh/GitHubAiAPI/StringManipulation.java

```diff
@@ -45,4 +53,11 @@ public static String findVowels(String input) {
         }
         return vowels.toString();
     }
+
+    // Helper method to check if a string is a palindrome
+    public static boolean isPalindrome(String input) {
+        String cleanInput = input.replaceAll("\\s+", "").toLowerCase(); // remove spaces and convert to lower case
```

Comment:
The isPalindrome method currently removes all spaces from the string before checking if it's a palindrome. This might not be the desired behavior in all cases. Consider adding a parameter to control whether spaces should be ignored.